.vscode/settings.json

```diff
@@ -257,5 +257,8 @@
 
 	"[css][html][markdown][yaml]": {
 		"editor.defaultFormatter": "esbenp.prettier-vscode"
+	},
+	"[typescriptreact]": {
+		"editor.defaultFormatter": "biomejs.biome"
```

Comment:
look a couple lines above, we already set this

Reply:
I don't know why my vscode didn't recognize that